hack/second-controller.py

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env python3
+# Author: Chmouel Boudjnah <chmouel@redhat.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import argparse
+import os
+import sys
+
+import yaml
+
+HELP_DESC = """This script will add a second controller to pipelines-as-code
+
+It's designed to be used like this:
+
+kubectl apply -f <(python3 yamlconvert.py LABEL)
```

Comment:
I think it has to be
```suggestion
kubectl apply -f <(python3 second-controller.py LABEL)
```

Reply:
what do you mean by proper name? your help fix, right?